src/styles/TimeIncrementSelector.scss

```diff
@@ -0,0 +1,36 @@
+.TimeIncrementSelector {
+  .time-selector-button {
+    background-color: #fff;
+    font-size: 14px;
+    color: #000;
+    border: none;
+    padding: 8px;
+  }
+  .time-selector-menu {
+    padding-top: 8px;
```

Comment:
Should we stick to using bootstrap's defined variables for padding and border radius?  If not, should we define these as separate variables so we can re-use them w/o copying the whole style to other locations?

Reply:
I think we should move all of this sass to tailwind eventually, but first need to get to a state where we can remove bootstrap utils since they overlap